scripts/changelog.js

```diff
@@ -148,22 +157,21 @@ ${printList(items.security)}
 
 ### Documentation
 
-${_.isEmpty(items.documentation) ? "" :
+${items.documentation.length === 0 ? "" :
```

Comment:
Why some checks are `length` now and others are `isEmpty`?

Reply:
When the thing being checked is an array, I switched from `_.isEmpty` to `.length === 0`, and when the thing being checked is an object, I switched from `_.isEmpty` to my own `_.isEmpty`.

I needed to check that things like `dependencies: { foo: [] }` (empty because all entries would have been filtered out) would still be considered empty, which I couldn't do with `_.isEmpty`. Then I figured it wasn't worth keeping it for arrays, since it's a simple `length` check.

Does that make sense? It is also entirely possible I missed something obvious when switching...